app/js/lib/foxbox-settings.js

```diff
@@ -32,6 +32,10 @@ export default class FoxboxSettings extends Model {
     });
   }
 
+  clear() {
+    storage.clear();
```

Comment:
We really just want to remove the items set here so that in the future this lib can be embedded in 3rd party projects without having side effects.
It's probably good enough for now, but I'd really add a comment like:
```js
// @todo Remove only the items set here.
```

Lastly, in an ideal world, we want to wrap it in a promise. Local storage is known for blocking the main thread because it's a synchronous API. With a promise we would get the benefit of a consistent API with `foxbox-db.js` too.